apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -224,51 +232,51 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
-              <FormField
-                control={form.control}
-                name={'name'}
-                render={({ field }) => (
-                  <FormItem>
-                    <FormLabel>Name</FormLabel>
-                    <FormControl>
+            <SaveFormContext.Provider value={value}>
+              <SidebarContent>
+                <FormField
+                  control={form.control}
+                  name={'name'}
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Name</FormLabel>
                       <InputField>
-                        <Input placeholder="Untitled" {...field} disabled={isReadOnly} />
+                        <FormControl>
+                          <Input
+                            placeholder="Untitled"
+                            {...field}
+                            disabled={isReadOnly}
+                            {...AUTOCOMPLETE_PASSWORD_MANAGERS_OFF}
+                          />
+                        </FormControl>
                       </InputField>
-                    </FormControl>
-                    <FormMessage />
-                  </FormItem>
-                )}
-              />
-              <FormField
-                control={form.control}
-                name={'stepId'}
-                render={({ field }) => (
-                  <FormItem>
-                    <FormLabel>Identifier</FormLabel>
-                    <FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name={'stepId'}
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Identifier</FormLabel>
                       <InputField className="flex overflow-hidden pr-0">
-                        <Input placeholder="Untitled" className="cursor-default" {...field} readOnly />
+                        <FormControl>
+                          <Input placeholder="Untitled" className="cursor-default" {...field} readOnly />
+                        </FormControl>
                         <CopyButton valueToCopy={field.value} size="input-right" />
                       </InputField>
-                    </FormControl>
-                    <FormMessage />
-                  </FormItem>
-                )}
-              />
-            </SidebarContent>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+              </SidebarContent>
+              <Separator />
 
-            {isInlineConfigurableStep && (
-              <>
-                <Separator />
-                <SidebarContent>
-                  <InlineControlValues />
-                </SidebarContent>
-              </>
-            )}
+              {isInlineConfigurableStep && <InlineControlValues />}
```

Comment:
the delay and digest inline control values components have the `SidebarContent` definition, otherwise we can't have many "sections" divided with "separators"